core/services/pipeline/common.go

```diff
@@ -55,6 +53,8 @@ type (
 var (
 	ErrWrongInputCardinality = errors.New("wrong number of task inputs")
 	ErrBadInput              = errors.New("bad input for task")
+	ErrParameterEmpty        = errors.New("parameter is empty")
+	ErrTooManyErrors         = errors.New("too many errors")
```

Comment:
Does it usually include the details of said errors?

Reply:
Yeah I've tried to be quite strict about wrapping these errors and using `errors.Cause()` in all checks, but if you spot somewhere I missed, please let me know